tests/utils/test_audio_utils.py

```diff
@@ -63,6 +67,10 @@ def test_mel_to_hertz(self):
         expected = np.array([60, 100, 200, 1000, 1001, 2000])
         self.assertTrue(np.allclose(mel_to_hertz(inputs, "slaney"), expected))
 
+        expected = np.array([60, 100, 200, 1000, 1001, 2000])
+        inputs = np.array([92.6824, 150.4899, 283.2313, 999.9907, 1000.6534, 1521.3674])
```

Comment:
(nit) let's swap these round for consistency
```suggestion
        inputs = np.array([92.6824, 150.4899, 283.2313, 999.9907, 1000.6534, 1521.3674])
        expected = np.array([60, 100, 200, 1000, 1001, 2000])
```